models/events.js

```diff
@@ -6,11 +6,12 @@ const Schema = mongoose.Schema;
 const states = ['Tercera edad', 'Entorno penitenciario', 'Salud mental', 'Convivencia', 'Hospitales', 'Menores', 'Discapacidad'];
 
 const EventsSchema = new Schema({
-    role: {type: String, enum: states},
-    name: {type: String, text: true},
+    role: {type: String, enum: states, required: true},
+    name: {type: String, text: true, required: true},
     description: {type: String, text: true},
     photo: String,
-    date: Date,
+    date: {type: Date, required: true},
+    uploadDate: { type: Date, default: Date.now()},
```

Comment:
Las funciones de default van sin parentesis
```suggestion
    uploadDate: { type: Date, default: Date.now},
```